portal-frontend/src/app/features/create-submission-dialog/create-submission-dialog.component.html

```diff
@@ -0,0 +1,150 @@
+        <mat-radio-group [value]="selectedSubmissionType?.code" (change)="onSubmissionTypeSelected($event)" color="primary">
+          <mat-radio-button
+            *ngFor="let subType of submissionTypes"
+            [value]="subType.code"
+            class="radio-option"
+            required
+          >
+            {{ subType.label }}
+          </mat-radio-button>
+        </mat-radio-group>
+      </div>
+      <div id="warningBanner" class="warning-banner" *ngIf="selectedSubmissionType">
+        <div>
+          <b class="sub-type-label">{{ selectedSubmissionType.label }}</b>
+          <div
+            class="typeDescription"
+            [ngClass]="{ 'ellipsis-3': !readMoreClicked }"
+            [innerHtml]="selectedSubmissionType.portalHtmlDescription"
+          ></div>
+        </div>
+      </div>
+      <span
+        class="read-more-btn"
+        [ngClass]="{ 'display-none': !isReadMoreVisible }"
+        *ngIf="selectedSubmissionType"
+        (click)="onReadMoreClicked()"
+        ><u>Read {{ readMoreClicked ? 'Less' : 'More' }}</u></span
+      >
+    </ng-container>
+    <ng-container *ngIf="currentStep === steps.applicationType">
+      <div class="step-description">Select an option to learn more about the application type.</div>
+      <div class="step-application-type">
+        <mat-radio-group
+          [value]="selectedAppType?.code"
+          class="app-type-radio-group"
+          aria-labelledby="radio-label"
+          (change)="onAppTypeSelected($event)"
+          color="primary"
+          required
+        >
+          <mat-radio-button *ngFor="let appType of applicationTypes" class="radio-option" [value]="appType.code">
+            {{ appType.portalLabel }}
+          </mat-radio-button>
+        </mat-radio-group>
+      </div>
+
+      <div id="warningBanner" class="warning-banner" *ngIf="selectedAppType">
+        <div>
+          <b class="app-type-label radio-option">{{ selectedAppType.portalLabel }}</b>
+        </div>
+        <div
+          class="typeDescription"
+          [ngClass]="{ 'ellipsis-3': !readMoreClicked }"
+          [innerHtml]="selectedAppType.htmlDescription"
+        ></div>
+        <span class="read-more-btn" [ngClass]="{ 'display-none': !isReadMoreVisible }" (click)="onReadMoreClicked()"
+          ><u>Read {{ readMoreClicked ? 'Less' : 'More' }}</u></span
+        >
+      </div>
+    </ng-container>
+    <ng-container *ngIf="currentStep === steps.noticeOfIntentType">
+      <div class="step-description">Select an option to learn more about the notice of intent type.</div>
+      <div class="step-noi-type">
+        <mat-radio-group
+          class="app-type-radio-group"
+          aria-labelledby="radio-label"
+          (change)="onNoiTypeSelected($event)"
+          color="primary"
+          required
+        >
+          <mat-radio-button *ngFor="let appType of noticeOfIntentTypes" class="radio-option" [value]="appType.code">
+            {{ appType.portalLabel }}
```

Comment:
Nitpick: `noiType` or just `type` instead of `appType`

Reply:
Done!